randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1971,17 +1971,23 @@ Extra - room_id: [42]
   > Event - Serris
       All of the following:
           Any of the following:
-              # Serris Weapon Requirements
-              Charge Beam or Missiles ≥ 10
-              Missiles ≥ 5 and 20 DMG Missiles
+              # Serris Weapon Requirements (50HP)
+              Charge Beam or Missiles ≥ 20
+              Missiles ≥ 10 and 20 DMG Missiles
+              All of the following:
+                  Combat (Advanced)
+                  Any of the following:
+                      Missiles ≥ 10
+                      Missiles ≥ 5 and 10 DMG Missiles
```

Comment:
missing 10 dmg missile, the second one should be 20 dmg missile

Reply:
this is for advanced logic, not missing any shots. Don't see why we we need it to be 10x10DMG and 5x20DMG >> 50HP.

Reply:
see above comment.